src/psyclone/psyir/nodes/acc_directives.py

```diff
@@ -743,45 +756,54 @@ class ACCUpdateDirective(ACCStandaloneDirective):
     It includes a direction attribute that can be set to 'self', 'host' or
     'device' and the symbol that is being updated.
 
-    :param symbol: the symbol to synchronise with the accelerator.
-    :type symbol: :py:class:`psyclone.psyir.symbols.DataSymbol`
+    :param signatures: the access signature(s) that need to be synchronised \
+                       with the accelerator.
+    :type signatures: Set[:py:class:`psyclone.core.Signature`]
```

Comment:
Why? I don't know, thoughts @arporter?
I never gave it much thought though, because, honestly, I didn't know better before I read your comment.
I think what you wrote makes sense and I'm guilty of never having read the dev guide top to bottom to have a full picture of what's happening...